tests/voq/test_fabric_reach.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import yaml
+
+logger = logging.getLogger(__name__)
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+supReferenceData = {}
+localModule = 0
+supervisorAsicBase = 300
```

Comment:
Why was this supervisorAsicBase hard-coded to 300. FYI, on Nokia chassis, the value of 1 for supervisorAsicBase works to get the 'Remote Module' in 'show fabric reachability', and not 300.

Reply:
This is the modid of the asic. For Arista systems, on the supervisor side, this starts from 300. 
This is set by soc property appl_param_module_id for each asic on the supervisor.